Data/ByteString.hs

```diff
@@ -752,6 +752,7 @@ scanr1 f ps
 --
 -- This implemenation uses @memset(3)@
 replicate :: Int -> Word8 -> ByteString
+{-# inline replicate #-}
```

Comment:
Should that `inline` be upper-case `INLINE`?  I've never seen the lower-case variant before...